Tests/CommandsTests/RunToolTests.swift

```diff
@@ -86,14 +86,15 @@ final class RunToolTests: XCTestCase {
         }
     }
 
-    // Verifies that giving several options still works.
-    func testSanitizeThreadAddress() throws {
+    // Verifies that sanitization works
+    func testSanitizeThread() throws {
         fixture(name: "Miscellaneous/SanitizersTest") { path in
-            let cmdline = ["--sanitize=thread", "--sanitize=address"]
+            let cmdline = ["--sanitize=thread"]
             XCTAssertThrows(try execute(cmdline, packagePath: path)) { (error: SwiftPMProductError) -> Bool in
                 switch error {
-                case .executionFailure:
-                    return true
+                case .executionFailure(_, _, let stderr):
+                    return stderr.range(of: "ThreadSanitizer:") != nil
+                        && stderr.range(of: "access race") != nil
```

Comment:
This runs just nice.

Reply:
This works but I am seeing a crash report generated after running this test. I am not sure if that is expected with the sanitizers.

Reply:
This report is not from the program itself, it is from some sort of wrapper. The program nicely `stderr`ors the thread sanitizer stuff out, and we can access it here.